pymc_experimental/model_builder.py

```diff
@@ -187,18 +187,15 @@ def load(cls, fname):
         filepath = Path(str(fname))
         data = az.from_netcdf(filepath)
         idata = data
-        # Since there is an issue with attrs getting saved in netcdf format which will be fixed in future the following part of code is commented
-        # Link of issue -> https://github.com/arviz-devs/arviz/issues/2109
-        # if model.idata.attrs is not None:
-        #     if model.idata.attrs['id'] == self.idata.attrs['id']:
-        #         self = model
-        #         self.idata = data
-        #         return self
-        #     else:
-        #         raise ValueError(
-        #             f"The route '{file}' does not contain an inference data of the same model '{self.__name__}'"
-        #         )
-        return idata
+        if idata.attrs is not None:
```

Comment:
if arviz has that patch let's remove this.

Reply:
it has but I added it to make sure one can't load a `.nc` file with no attrs, If it's not required lets just remove it

Reply:
I think we run into problems if we can't save attrs, we need it.